cirq-core/cirq/linalg/decompositions.py

```diff
@@ -71,7 +71,13 @@ def _rotation_matrix(angle: float) -> np.ndarray:
 
 
 def deconstruct_single_qubit_matrix_into_angles(mat: np.ndarray) -> Tuple[float, float, float]:
-    """Breaks down a 2x2 unitary into more useful ZYZ angle parameters.
+    r"""Breaks down a 2x2 unitary into more useful ZYZ angle parameters.
```

Comment:
I'd remove "more useful". It's judgmental and incorrect. Matrix is more useful in some cases and angles are more useful in others (try computing the trace from the angles).

Suggestion:

```
Breaks down a 2x2 unitary into ZYZ rotation angles.
```

Reply:
Done.